python/encode.py

```diff
@@ -0,0 +1,13 @@
+"""Script to encode a string as Base64"""
+import base64
+import sys
+
+test_string = sys.argv[1]
+
+# Encode string argument as bytes
+test_bytes = test_string.encode("utf-8")
+
+encoded_bytes = base64.b64encode(test_bytes)
+
+# Print as a string, not bytes
+print(encoded_bytes.decode("utf-8"))
```

Comment:
I don't know if this is overcomplicating things to print as a string rather than bytes, but otherwise the docker container log will show "b'b'Hello, world!'\n'\n" which is annoying

Reply:
I thought it must be over-complicated, but I dove in and it seems like the `.encode()` and `.decode()` are indeed necessary. 🤷

The one thing is that the `"utf-8"` parameter to `.encode()` and `.decode()` is the default so we can omit it.